pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -377,8 +377,8 @@ func (m *managerImpl) synchronize(diskInfoProvider DiskInfoProvider, podFunc Act
 		// If the pod is marked as critical and static, and support for critical pod annotations is enabled,
```

Comment:
comments also need some update, we will now do not evict static *or* critical pod.